packages/prisma/schema.prisma

```diff
@@ -643,6 +653,11 @@ enum WorkflowMethods {
   SMS
 }
 
+enum DeploymentLicenseType {
+  FREE
+  EE
+}
+
```

Comment:
Can we split these migrations in a prep PR?

Reply:
Do self hosters really care about these models?

Reply:
That's another good point @emrysal we don't want to mixup what should live in our console DB with here. We can also check for `CALCOM_LICENSE_KEY` for this.

Reply:
The idea was to move out from environment variable, to let admins setup their instances as much as we can from the admin wizard, like we did with activating apps from it.

Reply:
@zomars I'm afraid that doing a prep PR with the migrations separately could introduce churn on why was the new model was introduced. Any particular reason for doing it? Just to understand the rationale, learn and do it differently next time.

Reply:
Also we can assume that an empty key is a "FREE" license.